samples/tutorial/tutorial-2-complete/src/main/java/workflow/tutorial/TutorialActivity.kt

```diff
@@ -0,0 +1,30 @@
+@file:OptIn(WorkflowUiExperimentalApi::class)
+
+package workflow.tutorial
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.squareup.workflow1.ui.ViewRegistry
+import com.squareup.workflow1.ui.WorkflowRunner.Config
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.backstack.BackStackContainer
+import com.squareup.workflow1.ui.setContentWorkflow
+
+private val viewRegistry = ViewRegistry(
+    BackStackContainer,
```

Comment:
I think an explicit step above when bringing in BackStackScreen that we need to add the `BackStackContainer` to the ViewRegistry would be helpful.